bin/init.bash

```diff
@@ -79,6 +83,10 @@ elif [[ "${flavor}" == "gcp" ]]; then
   cp -r "${config_defaults_path}/gcp/group_vars" "${config_path}/"
 elif [[ "${flavor}" == "aws" ]]; then
   cp -r "${config_defaults_path}/aws/group_vars" "${config_path}/"
+elif [[ "${flavor}" == "openstack" ]]; then
+  cp -r "${config_defaults_path}/openstack/group_vars" "${config_path}/"
+elif [[ "${flavor}" == "citycloud" ]]; then
+  cp -r "${config_defaults_path}/openstack/group_vars" "${config_path}/"
```

Comment:
Hmm, would that work. Since when we copy these files the structure changes so the links would need to change. 

If I understand you correctly the structure would look something like this:
```
openstack/group_vars/some_openstack_file.txt
citycloud/group_vars/some_citycloud_file.txt -> ../../openstack/group_vars/some_openstack_file.txt
```
And after you copied the citycloud config it would look like this:
```
group_vars/some_citycloud_file.txt -> ../../openstack/group_vars/some_openstack_file.txt
```
Or am I missing something?

Reply:
Nice catch @Xartos ! I think you would need to use `cp --dereference`. I think this is already done some in other parts of kubespray.

Reply:
Right, yea then I agree. That is a quite neat solution

Reply:
Actually, I think that this change might go a bit too far for this issue. I'll create a new issue for this instead.